src/extension/app-schema/app-schema-test/pom.xml

```diff
@@ -163,6 +163,21 @@
             <version>2.0.0-1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+	     	<groupId>com.fasterxml.jackson.core</groupId>
+	     	<artifactId>jackson-core</artifactId>
+	     	<scope>test</scope>
+	   	</dependency>
+	   	<dependency>
+		    <groupId>com.fasterxml.jackson.core</groupId>
+		    <artifactId>jackson-annotations</artifactId>
+		    <scope>test</scope>
+	    </dependency>
```

Comment:
Same here :scream_cat:.

Reply:
Looks like this is the last one, good I run out of screaming icons :relieved:.

Reply:
fixed